### ISSUE TITLE
altitude-offset not work with terrain-gpu

### ISSUE THREAD

**Issue by remoe** [gwaldron/osgearth#1184]
The second layer "world_boundaries2" should fly over 10'000 m (master):

```xml
<!--
osgEarth Sample
-->

<map name="Worldwide Line Vectors">

    <image name="readymap_imagery" driver="tms">
        <url>http://readymap.org/readymap/tiles/1.0.0/7/</url>
    </image>

    <elevation name="readymap_elevation" driver="tms" vdatum="egm96">
        <url>http://readymap.org/readymap/tiles/1.0.0/116/</url>
    </elevation>
            
    <feature_source name="world-data" driver="ogr">
        <url>../data/world.shp</url>
        <convert type="line"/>
    </feature_source>
    
    <feature_model name="world_boundaries" feature_source="world-data">
        
        <layout tile_size="500000" crop_features="true" paged="true">
            <level max_range="1e10"/>
        </layout>
                
        <styles>
            <style type="text/css">
                world {
                   stroke:                   #ffff00;
                   stroke-width:             1000m;
                   stroke-tessellation-size: 1km;
                   render-lighting:          false;
                   altitude-clamping:        terrain-gpu;
                   render-depth-offset:      true;                   
                }            
            </style>
        </styles>
        
    </feature_model>

    <feature_model name="world_boundaries2" feature_source="world-data">
        
        <layout tile_size="500000" crop_features="true" paged="true">
            <level max_range="1e10"/>
        </layout>
                
        <styles>
            <style type="text/css">
                world {
                   stroke:                   #0000ff;
                   stroke-width:             500m;
                   stroke-tessellation-size: 1km;
                   render-lighting:          false;
                   altitude-clamping:        terrain-gpu;
                   render-depth-offset:      true;     
                   altitude-offset: 10000m;              
                }            
            </style>
        </styles>
        
    </feature_model>
</map>
```

**Comment by jasonbeverage** [comment]
Hey Remo,

This should be fixed in 78f3cb0b0d0037e9cf5ede79378dea91981b857b.

Thanks!

Jason

**Comment by remoe** [comment]
Thanks Jason! Works!
